src/Components/Steps/Zipcode.tsx

```diff
@@ -128,6 +139,20 @@ export const Zipcode = () => {
       <QuestionQuestion>
         <FormattedMessage id="questions.zipcode" defaultMessage="What is your zip code?" />
       </QuestionQuestion>
+      {!noChangeStateMessage && (
```

Comment:
You mentioned in the PR description to wait to merge this, but since this is in a feature flag couldn't we go ahead and merge but have it off?